README.md

```diff
@@ -171,344 +54,31 @@ The following guide assumes you have some sort of ES2015 build set up using babe
 You can also start with [preact-boilerplate] or a [CodePen Template](http://codepen.io/developit/pen/pgaROe?editors=0010).
```

Comment:
I think it is better to replace `preact-boilerplate` to `preact-cli`.

Reply:
Preact-cli is mentioned above and underneath

Reply:
`preact-boilerplate` recommends to use `preact-cli`.
I think it may be better to remove this line.
https://github.com/developit/preact-boilerplate#preact-boilerplate--starter-kit--

Reply:
Good spot!